sechub-shared-kernel/src/main/java/com/mercedesbenz/sechub/sharedkernel/shutdown/SpringApplicationShutdownHandler.java

```diff
@@ -33,14 +32,13 @@ public void register(ShutdownListener shutdownListener) {
         shutdownListeners.add(shutdownListener);
     }
 
-    @Override
-    public void onApplicationEvent(ContextClosedEvent event) {
-        log.info("Spring application context closed - informing shutdown listeners");
+    public void handleShutdown() {
+        log.info("Start handling shutdown - informing shutdown listeners");
         shutdownListeners.forEach(listener -> {
             try {
                 listener.onShutdown();
-            } catch (Exception e) {
-                log.error("Failed to notify shutdown listener {}", listener.getClass(), e);
+            } catch (RuntimeException e) {
+                log.error("Notified shutdown listener failed: {}", listener.getClass(), e);
```

Comment:
unclear wording.
Maybe one of these:
"Shutdown of listener {} failed"
"Shutdown notification of listener {} failed"